backends/ebpf/ebpfTable.cpp

```diff
@@ -363,7 +363,7 @@ void EBPFTable::emitInitializer(CodeBuilder* builder) {
 
     for (auto p : *mcd.substitution.getParametersInArgumentOrder()) {
         builder->emitIndent();
-        builder->appendFormat(".%s = ", p->name.name.c_str());
+        builder->appendFormat(".u = ", p->name.name.c_str());
```

Comment:
This cannot be correct. It's either u with no argument or %s with an argument or u.%s with an argument.

Reply:
Yes, sorry about that. I actually wanted to discuss this change too, I have not tested it with multiple action arguments yet. I am going to look into that.